components/org.wso2.carbon.metrics.manager/src/main/java/org/wso2/carbon/metrics/manager/MetricHierarchy.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@prabathabey It was already been fixed in [commit:0402c09](https://github.com/wso2/carbon-metrics/commit/d8ad48ed53fd2c90ebcfe5923a3ae3e00215bc43).